ietf/templates/meeting/agenda.html

```diff
@@ -102,6 +102,11 @@ <h2>
     <h2 class="mt-3">
         {% if personalize %}Personalize{% endif %}
         Detailed Agenda
+        {% if schedule.meeting.agenda_warning_note %}
+            <span class="badge rounded-pill bg-danger">
+              {{ schedule.meeting.agenda_warning_note|removetags:"h1" |safe }}
```

Comment:
Is this "no h1 but everything else is safe" dance reintroduction of legacy code?
I wonder if the right thing to do, instead, is to make agenda_warning_note be either plain text (never marked safe) or be made markdown (migrating any old content, though such probably shouldn't exist at all).

Reply:
It's copy-pasted verbatim from the instance of the flag that was left on the week-view header. This note really needs to be plaintext - it's a badge on a page heading, so more than a word or two is a recipe for yuck.

Reply:
Just checked, and aside from my own debugging there are no non-empty `agenda_warning_notes` in the dev DB.

Reply:
per further discussion, we'll turn this into a boolean in the long run, but we can work towards bringing the fix in this PR in for now.